testing/endtoend/endtoend_test.go

```diff
@@ -234,20 +231,27 @@ func (r *testRunner) testTxGeneration(ctx context.Context, g *errgroup.Group, ke
 	})
 }
 
-func (r *testRunner) waitForMatchingHead(ctx context.Context, timeout time.Duration, check, ref *grpc.ClientConn) error {
+func (r *testRunner) waitForMatchingHead(ctx context.Context, timeout time.Duration) error {
```

Comment:
just wanted to make sure has this been the starndard approach in previous PRs to deal with e2e.

Reply:
I went through it with Raul as I was scared to touch the e2e without being sure of what I was doing. I would be more than happy to get someone else to double check it as well just to be certain.